### ISSUE TITLE
Text not updated after pause/resume when using WebView on iOS

### ISSUE THREAD

**Issue by suhlatwork** [flutter/flutter#28471]
I am trying to update the title in the app bar of a scaffold via `setState`. On iOS, this is working fine until the app is paused and resumed (i.e. press the home button, and then switch back to the app). After the pause/resume, the title itself is no longer updated. I made sure `setState` is still called; the variable is indeed changed, the `build` method sees the new value, and even `debugDumpApp ()` shows the new value. This appears to only happen if a `WebView` widget (from package `webview_flutter`) is used. It seems that not only the title, but also any other text widget is affected. This did not happen for my tests on Android. There are no messages, error messages, or exceptions.

## Steps to Reproduce

```dart
import 'package:flutter/material.dart';
import 'package:webview_flutter/webview_flutter.dart';

void main() => runApp(MaterialApp(
      title: 'Flutter Demo',
      home: MyHomePage(),
    ));

class MyHomePage extends StatefulWidget {
  MyHomePage({Key key}) : super(key: key);

  @override
  _MyHomePageState createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  int _counter = 0;

  void _incrementCounter() {
    setState(() {
      _counter++;
    });
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('$_counter'),
      ),
      body: Center(
        child: WebView(
          initialUrl: 'https://flutter.io',
        ),
      ),
      floatingActionButton: FloatingActionButton(
        onPressed: _incrementCounter,
        tooltip: 'Increment',
        child: Icon(Icons.add),
      ), // This trailing comma makes auto-formatting nicer for build methods.
    );
  }
}
```

1. start the app from above
2. click the floating action button a couple of times, the number in the app bar should increase accordingly
3. pause and resume the app by pressing the home button and then returning to the app
4. click the floating action button, the number in the app bar is no longer increased despite the `_incrementCounter` method still being called

## Logs

There are no messages after the start of the app.

`flutter analyze` does not find any issue.

```
$ flutter doctor -v
[✓] Flutter (Channel stable, v1.0.0, on Mac OS X 10.13.6 17G5019, locale de-DE)
    • Flutter version 1.0.0 at /Users/sebastian/Documents/Software/flutter
    • Framework revision 5391447fae (3 months ago), 2018-11-29 19:41:26 -0800
    • Engine revision 7375a0f414
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at /Users/sebastian/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    • ios-deploy 1.9.4
    • CocoaPods version 1.6.1

[✓] Android Studio (version 3.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 33.1.1
    • Dart plugin version 182.5215
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)

[✓] Connected device (2 available)
    • XT1068               • XX                               • android-arm • Android 6.0 (API 23)
    • iPhone von XX • XX • ios         • iOS 12.1.4
```

**Comment by zoechi** [comment]
I couldn't reproduce with `webview_flutter` `0.3.3+1` 
```
[✓] Flutter (Channel master, v1.2.3-pre.47, on Mac OS X 10.14.3 18D42, locale en-AT)
[✓] Connected device (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 9 (API 28) (emulator)
```

**Comment by suhlatwork** [comment]
I tried a bit more:
* I cannot reproduce in a simulator, only on the real device
* updating `webview_flutter` did not help
* updating Flutter fixes the issue, the fix seems to be introduced between v1.1.4 and v1.1.5

-> might be a duplicate of issue #24900

**Comment by zoechi** [comment]
> the workaround then seems to be to use the beta channel,

Right. Stable is quite outdated, but `beta` is rather new.
